roles/docker/tasks/systemcontainer_docker.yml

```diff
@@ -170,6 +170,18 @@
   retries: 3
   delay: 30
 
+- name: Check for credentials file for registry auth
+  stat:
+    path: "{{ docker_cli_auth_config_path }}/config.json"
+  when: oreg_auth_user is defined
+  register: docker_cli_auth_credentials_stat
+
+- name: Create credentials for docker cli registry auth
+  command: "docker --config={{ docker_cli_auth_config_path }} login -u {{ oreg_auth_user }} -p {{ oreg_auth_password }} {{ oreg_host }}"
+  when:
+    - oreg_auth_user is defined
+    - (not docker_cli_auth_credentials_stat.stat.exists or oreg_auth_credentials_replace) | bool
+
 - set_fact:
```

Comment:
`roles/docker/tasks/registry_auth.yml` is doing this already, possibly it's better to `include` it directly.

Reply:
I've pushed a new version where this code is in common with `roles/docker/tasks/package_docker.yml`